packages/@webex/plugin-people/test/integration/spec/people.js

```diff
@@ -97,7 +97,7 @@ describe('plugin-people', function () {
           assert.equal(person.emails[0], mccoy.email);
         }));
 
-      it('returns a list of people matching name', () =>
+      it.only('returns a list of people matching name', () =>
```

Comment:
@sreenara, why we are running only this test, should we remove only from this or this in intentional ?